StxInsider/app/src/main/kotlin/com/stxnext/stxinsider/util/Util.kt

```diff
@@ -150,6 +151,26 @@ fun Activity.bindKViews() {
         }
 }
 
+fun Int.colorDarker(factor: Float): Int {
```

Comment:
Please change name of this method, because for some `factor` values it will create lighter color.